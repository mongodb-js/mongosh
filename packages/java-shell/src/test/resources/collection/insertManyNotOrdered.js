// before
db.coll.deleteMany({});
// command
db.coll.insertOne({_id: "b", name: "b"});
db.coll.insertMany([{_id: "b", name: "b"}, {_id: "d", name: "d"}], {ordered: false});
// command
db.coll.find();
// clear
db.coll.drop();
