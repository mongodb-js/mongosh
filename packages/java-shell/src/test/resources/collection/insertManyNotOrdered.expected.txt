com.mongodb.MongoBulkWriteException: Bulk write operation error on server %mongohostport%. Write errors: [BulkWriteError{index=0, code=11000, message='E11000 duplicate key error collection: admin.coll index: _id_ dup key: { _id: "b" }', details={}}].
[ { "_id": "b", "name": "b" }, { "_id": "d", "name": "d" } ]